lib/Completion/Bridge/TolerantParser/CompletionContext.php

```diff
@@ -264,6 +264,11 @@ public static function anonymousUse(Node $node): bool
 
     public static function methodName(Node $node): bool
     {
+        // TODO better distinguish empty body from empty method parameter list
```

Comment:
can we remove TODO and instead document what this condition is for? TODO will never get done.

Reply:
Done :wink: